src/views/Dashboard.js

```diff
@@ -114,6 +114,14 @@ const Dashboard = () => {
     }
   }, [location?.state]);
 
+  useEffect(() => {
```

Comment:
This code is on 'main' branch... I am not sure why it here. Looks like development isn't sync with main maybe properly?